gr-digital/python/digital/ofdm_sync_pn.py

```diff
@@ -39,7 +39,7 @@ def __init__(self, fft_length, cp_length, logging=False):
         """
         OFDM synchronization using PN Correlation:
         T. M. Schmidl and D. C. Cox, "Robust Frequency and Timing
-        Synchonization for OFDM," IEEE Trans. Communications, vol. 45,
+        Synchronization for OFDM," IEEE Trans. Communications, vol. 45,
```

Comment:
Heh, nice, this must have happened when someone typed that paper title from the back of his head :) funny.

Reply:
😆